### PR TITLE
Migrate Documentation to Antora

### PR DESCRIPTION
**Pull request by settermjd**
This PR migrates the existing documentation to Antora, just like the iOS documentation [is being migrated](https://github.com/owncloud/ios/pull/1084).

### CODE REVIEW

**Comment by michaelstingl** [comment]
Is there a preview available somewhere?

**Comment by settermjd** [comment]
@michaelstingl, working on that atm.

**Comment by settermjd** [comment]
Here are some screenshots for you:

bildschirmfoto 2018-09-27 um 11 41 46
bildschirmfoto 2018-09-27 um 11 40 52
bildschirmfoto 2018-09-27 um 11 41 02

File: docs/antora.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: android
+title: The Android App
```
Comment (tboerger) [inline docs/antora.yml]: Maybe `Android Application`?

**Comment by settermjd** [comment]
Sure can, @davigonz.

File: docs/antora.yml (hunk 1 above)
Comment (settermjd) [reply docs/antora.yml]: Not sure, @tboerger. Would it make much of a difference eitherway?

Comment (tboerger) [reply docs/antora.yml]: Just thinking about how this looks like on the docs ;)

**Comment by davigonz** [comment]
Is there any update on this?

File: docs/antora.yml (hunk 1 above)
Comment (settermjd) [reply docs/antora.yml]: The current docs use this naming style, so I'm leaving it as is.

**Comment by settermjd** [comment]
@davigonz, can you review the linked PRs. All have been updated and need a review.

**Comment by davigonz** [comment]
> can you review the linked PRs. All have been updated and need a review.

Thanks @settermjd for updating the PRs, We have a couple of versions more in the market after 2.8 such as 2.9.0, 2.9.1, 2.9.2 and 2.9.3, have a look at https://github.com/owncloud/android/blob/master/CHANGELOG.md. Once those versions changes are included, I guess we can merge.

**Comment by tboerger** [comment]
We should update the branch and finally merge it since we are already using the new documentation deployment.

**Comment by davigonz** [comment]
> We should update the branch and finally merge it since we are already using the new documentation deployment.

Please rebase, fix the conflicts and we will merge it.

**Comment by settermjd** [comment]
@davigonz, done. Ready for merge.

**Comment by davigonz** [comment]
@settermjd merged

**Pull request merged**